xml/System.Net.NetworkInformation/PhysicalAddress.xml

```diff
@@ -329,16 +329,20 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The `address` parameter must contain a string that can only consist of numbers and upper-case letters as hexadecimal digits. Some examples of string formats that are acceptable are as follows:  
+ The `address` parameter must contain a string that can only consist of numbers and letters as hexadecimal digits. Some examples of string formats that are acceptable are as follows:  
   
  `001122334455`  
   
  `00-11-22-33-44-55`  
+
+ `0011.2233.4455`
+
+ `00:11:22:33:44:55`
   
  `F0-E1-D2-C3-B4-A5`  
-  
- Note that an address that contains `f0-e1-d2-c3-b4-a5` will fail to parse and throw an exception.  
-  
+
+ `f0-e1-d2-c3-b4-a5`
```

Comment:
```suggestion
 `f0-e1-d2-c3-b4-a5` (.NET 5 and later versions only)
```

Reply:
Same as above - doesn't seem necessary, but let me know if that's incorrect.